progressbar_test.go

```diff
@@ -26,3 +26,14 @@ func TestBar(t *testing.T) {
 		t.Error("should have an error for adding > bar")
 	}
 }
+
+func ExampleProgressBar_RenderBlank() {
+	bar := New(10)
+	bar.SetSize(10)
+	bar.Add(5)
```

Comment:
`bar.Add(5)` will create a bar. If you only want to test the one bar, then I would remove this. Otherwise your output should be expected to be:

```
50% |█████     | [0s:0s]            
   0% |          | [0s:0s]
```

Reply:
Should `.Reset()` clear the output? (Can it actually clear it.. ?)